evalml/pipelines/component_graph.py

```diff
@@ -691,13 +697,18 @@ def describe(self, return_dict=False):
         for number, component in enumerate(self.component_instances.values(), 1):
             component_string = str(number) + ". " + component.name
             logger.info(component_string)
-            components.update(
-                {
-                    component.name: component.describe(
-                        print_name=False, return_dict=return_dict
-                    )
-                }
-            )
+            try:
+                components.update(
+                    {
+                        component.name: component.describe(
+                            print_name=False, return_dict=return_dict
+                        )
+                    }
+                )
+            except TypeError:
+                raise TypeError(
+                    "You need to instantiate ComponentGraph before calling describe()"
+                ) from None
```

Comment:
do we need this `None` here? If not we can remove it!

Reply:
If I don't include `from None` it spits out 2 error messages rather than one weirdly enough (I'm not entirely sure why it does that).

Reply:
can you show me the stack trace if you have it?